README.md

```diff
@@ -1,7 +1,7 @@
 # Hello World: Introduction and F´ Terminology
 
-> Note: Tutorials can be browsed through our website or through the GitHub repository:
-> -> [Go to the website](https://fprime-community.github.io/fprime-tutorial-hello-world)
+> Note: Tutorials can be browsed through our website or through the GitHub repository:\
```

Comment:
Is the \ after repository supposed to be there?

Reply:
It forces the linebreak to be rendered